manifest-production.yml

```diff
@@ -1,13 +1,13 @@
 ---
-host: service-manual
+host: guides
 buildpack: nodejs_buildpack
 memory: 256M
-instances: 1
+instances: 3
```

Comment:
Curious why we use 3 instances? I've never seen more than 1 in use.

Reply:
i want more than 1 for H/A if we lost a diego-cell.  as for why 3 instead of 1....  smart-cities is using 3 and 256M of memory means i'm not concerned about resources